Pull Request #95 broke travis-ci builds

The build log from travis-ci shows the build breaking with:
```
gcc -pthread -fno-strict-aliasing -g -fstack-protector --param=ssp-buffer-size=4 -Wformat -Werror=format-security -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I./port3 -I/opt/python/2.7.14/include/python2.7 -c bluez/btmodule.c -o build/temp.linux-x86_64-2.7/bluez/btmodule.o
bluez/btmodule.c: In function ‘init_bluetooth’:
bluez/btmodule.c:3688:22: error: ‘HCI_CHANNEL_USER’ undeclared (first use in this function)
     ADD_INT_CONST(m, HCI_CHANNEL_USER);
                      ^
bluez/btmodule.c:3069:60: note: in definition of macro ‘ADD_INT_CONST’
 #define ADD_INT_CONST(m, a) PyModule_AddIntConstant(m, #a, a)
                                                            ^
bluez/btmodule.c:3688:22: note: each undeclared identifier is reported only once for each function it appears in
     ADD_INT_CONST(m, HCI_CHANNEL_USER);
                      ^
bluez/btmodule.c:3069:60: note: in definition of macro ‘ADD_INT_CONST’
 #define ADD_INT_CONST(m, a) PyModule_AddIntConstant(m, #a, a)
                                                            ^
error: command 'gcc' failed with exit status 1
```

This is because the we are build with a really old version of bluez (4.101) which was released 6 years ago.
This pull request leverages the bluez User Channel that was added about 4 years ago:

https://git.kernel.org/pub/scm/bluetooth/bluez.git/commit/lib/hci.h?id=c0b7506c42f73a5ad2d4d05c14b2676e6a08edb9

If we can update the travis config to use at least `xenial` instead of `trusty` I think this will just work.

--- a/.travis.yml
+++ b/.travis.yml
@@ -4,13 +4,22 @@ python:
   - "2.7"
   - "3.6"
 
+env:
+  global:
+    - BLUEZ_SRC_DIR="./contrib/bluez"
+    - LOCAL_INCLUDE_DIR="./include"
+
 os:
   - linux
 
 before_install:
+  
+  - git clone https://git.kernel.org/pub/scm/bluetooth/bluez.git -n $BLUEZ_SRC_DIR && cd $BLUEZ_SRC_DIR && git checkout $(git describe --tags $(git rev-list --tags='[0-9]*.[0-9]*' -n 1)) && cd -
+  - mkdir -p $LOCAL_INCLUDE_DIR/bluetooth && cp $BLUEZ_SRC_DIR/lib/*.h $LOCAL_INCLUDE_DIR/bluetooth
   - pip install -r requirements.txt
 
 install:
+  - CFLAGS="-I$LOCAL_INCLUDE_DIR" python setup.py build
   - python setup.py install
 
 script:

